.jenkins/tests/pom.xml

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.kie.jenkins</groupId>
+    <artifactId>jenkins-tests</artifactId>
+    <name>Kogito :: Jenkins Spock Tests</name>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <repositories>
+        <repository>
+            <id>jboss-public-repository-group</id>
+            <name>JBoss Public Repository Group</name>
+            <url>https://repository.jboss.org/nexus/content/groups/public/</url>
+            <layout>default</layout>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+            </snapshots>
+        </repository>
+        <repository>
+            <id>jenkins-releases</id>
+            <name>Jenkins Releases</name>
+            <url>https://repo.jenkins-ci.org/releases</url>
+        </repository>
+    </repositories>
+
+    <properties>
+        <!-- Do not deploy this artifact -->
+        <maven.deploy.skip>true</maven.deploy.skip>
+
+        <groovy.core.version>2.4.11</groovy.core.version>
```

Comment:
The same suggestion as here https://github.com/kiegroup/kogito-pipelines/pull/91#discussion_r537371129.